src/main/resources/application.properties

```diff
@@ -1,30 +1,10 @@
-spring.application.name=kniznica
-# Datab�zov� konfigur�cia
 spring.datasource.url=jdbc:mysql://localhost:3306/kniznica_final_bc_sda?createDatabaseIfNotExist=true
-# ?createDatabaseIfNotExist=true
-# &useSSL=false&serverTimezone=UTC
-spring.datasource.username=${user}
-spring.datasource.password=${password}
+spring.datasource.username=root
+spring.datasource.password=Marrri111
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 
-# Hibernate konfigur�cia
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 
-# Mo?nosti ?asovej z�ny
-spring.datasource.hikari.connection-timeout=20000
-spring.datasource.hikari.maximum-pool-size=5
-
-# Vypisovanie SQL pr�kazov do konzoly
-logging.level.org.hibernate.SQL=DEBUG
-logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
-
-# Zmena banneru
-spring.banner.location=classpath:custom-banner.txt
-
-# Lokaliz�cia SQL s�borov migr�ci� pre Flyway
-spring.flyway.locations=classpath:db/migration
-
-#server.port=8083
-
-
+spring.jpa.properties.hibernate.format_sql=true
```

Comment:
Slovak/Czech language is not the best choice but what can I do :D but if you wil use characters as ťčžý etc, than I could be problem.